event.tsx

```diff
@@ -16,3 +17,12 @@ export default function Event({ event }: { event: AppBooking }) {
     </div>
   );
 }
+
+const getEventStyle = (event: AppBooking) => {
+  if (event.floor.includes('1')) {
```

Comment:
Is this equivalent to `event.floor == "1"`? What is the content of the floor string?

Reply:
No here event.floor is a string, includes checks the value '1' is anywhere in the string
== would check that event.floor equals to '1'